manage_projects.rb

```diff
@@ -523,7 +523,7 @@ def add_budget(project)
   while !valid_date
     print "Effective at (YYYY-MM-DD): "
     valid_date = begin
-      Date.parse(gets.chomp)
+      Date.parse(gets.chomp).strip
```

Comment:
This and the similar one on line 323 need to have the `.strip` inside the brackets (or will break).